src/layers.js

```diff
@@ -146,8 +146,7 @@ const alignLayers = (
   );
 };
 
-// TODO: Rename this getSelectionBounds?
```

Comment:
Sorry! This comment was misleading. 😬 It would've made sense when I wrote it, but I don't think it makes sense anymore. The `getLayerBounds` function has no knowledge of the selection, so it's name is fine (I assume at the time I wrote this comment it when automatically detecting the selection or something like that).

Let's revert this commit.